library/python

```diff
@@ -90,23 +90,23 @@ GitCommit: 855b85c8309e925814dfa97d61310080dcd08db6
 Directory: 3.5/windows/windowsservercore
 Constraints: windowsservercore
 
-Tags: 3.6.0b1, 3.6
-GitCommit: a70c6819aa38d938d56e96408aa6e7c3a9162c64
+Tags: 3.6.0b2, 3.6
+GitCommit: aa533a4879c13944d62a5029929ffae78075d212
 Directory: 3.6
 
-Tags: 3.6.0b1-slim, 3.6-slim
-GitCommit: a70c6819aa38d938d56e96408aa6e7c3a9162c64
+Tags: 3.6.0b2-slim, 3.6-slim
+GitCommit: aa533a4879c13944d62a5029929ffae78075d212
 Directory: 3.6/slim
 
-Tags: 3.6.0b1-alpine, 3.6-alpine
-GitCommit: a70c6819aa38d938d56e96408aa6e7c3a9162c64
+Tags: 3.6.0b2-alpine, 3.6-alpine
+GitCommit: aa533a4879c13944d62a5029929ffae78075d212
 Directory: 3.6/alpine
 
-Tags: 3.6.0b1-onbuild, 3.6-onbuild
-GitCommit: 635ea5d58b53d165f7bedae90f8933c720a58150
+Tags: 3.6.0b2-onbuild, 3.6-onbuild
+GitCommit: aa533a4879c13944d62a5029929ffae78075d212
 Directory: 3.6/onbuild
 
-Tags: 3.6.0a4-windowsservercore, 3.6-windowsservercore
-GitCommit: 855b85c8309e925814dfa97d61310080dcd08db6
+Tags: 3.6.0b2-windowsservercore, 3.6-windowsservercore
+GitCommit: aa533a4879c13944d62a5029929ffae78075d212
```

Comment:
We actually need to revert the `windowsservercore` update back to 3.6.0a4 thanks to https://github.com/docker-library/python/issues/146 :cry: